docker/utils/utils.py

```diff
@@ -715,6 +715,27 @@ def create_host_config(binds=None, port_bindings=None, lxc_conf=None,
     return host_config
 
 
+def create_networking_config(endpoints_config=None):
+    networking_config = {}
+
+    if endpoints_config:
+        networking_config["EndpointsConfig"] = endpoints_config
+
+    return networking_config
+
+
+def create_endpoint_config(version,
+                           aliases=None):
+    endpoint_config = {}
+
+    if aliases:
+        if version_lt(version, '1.22'):
+            raise host_config_version_error('aliases', '1.22')
```

Comment:
Yeah. Perhaps just `endpoint_config.aliases`, since the user will have explicitly called `create_endpoint_config`. I'll update it.